### PR TITLE
Issue 9534: Add no-ship acme feature skeleton project

### LINKED ISSUE #9534: Re-introduce change reverted from 9160 -- No ship acmeCA feature
Re-introduce change reverted from https://github.com/OpenLiberty/open-liberty/pull/9160

Redeliver the no ship acmeCA feature skeleton project.

Under feature #9017

### PR DESCRIPTION
**Pull request by kristip17**
This reverts commit 6980c5d98d59bfce0b69f8ba05fde530f0773bdd.

Fixes #9534 

Re-delivery of #9160 

After design review comments, updated the feature name from acme to acmeCA.

The failure in the static build that caused the revert was having references to bcprov/bcpkix in the .feature file. Those have been removed.

### CODE REVIEW

**Comment by kristip17** [comment]
#build

**Comment by kristip17** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/acme-1.0/resources/l10n/com.ibm.websphere.appserver.acme-1.0.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides the ability to start the server and have it retrieve a CA signed server certificate using ACME.
```
[the remark this answers is not in the document]
Comment (kristip17) [reply dev/com.ibm.websphere.appserver.features/visibility/public/acme-1.0/resources/l10n/com.ibm.websphere.appserver.acme-1.0.properties]: I swapping things around a bit -- how are the changes?

**Comment by kristip17** [comment]
#run-libby-bot

File: dev/com.ibm.websphere.appserver.features/visibility/public/acme-1.0/resources/l10n/com.ibm.websphere.appserver.acme-1.0.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmuelle) [reply dev/com.ibm.websphere.appserver.features/visibility/public/acme-1.0/resources/l10n/com.ibm.websphere.appserver.acme-1.0.properties]: Looks good

**Comment by kristip17** [comment]
Splitting up the delivery of the project and feature files into two PRs.

See #9840 and #9856

**Comment by kristip17** [comment]
Delivered changes in other PRs.

**Pull request closed without merging**